pkg/collector/corechecks/cluster/kubernetes_apiserver.go

```diff
@@ -31,11 +31,12 @@ import (
 
 // Covers the Control Plane service check and the in memory pod metadata.
 const (
-	KubeControlPaneCheck         = "kube_apiserver_controlplane.up"
-	kubernetesAPIServerCheckName = "kubernetes_apiserver"
-	eventTokenKey                = "event"
-	maxEventCardinality          = 300
-	defaultResyncPeriodInSecond  = 300
+	KubeControlPaneCheck          = "kube_apiserver_controlplane.up"
+	kubernetesAPIServerCheckName  = "kubernetes_apiserver"
+	eventTokenKey                 = "event"
+	maxEventCardinality           = 300
+	defaultResyncPeriodInSecond   = 300
+	defaultTimeoutEventCollection = 1000
```

Comment:
could we set it a 2000ms here and remote the `*2` in the ticker creation

```suggestion
	defaultTimeoutEventCollection = 2000
```

Reply:
ok